source/install/prod-windows-2012.rst

```diff
@@ -0,0 +1,419 @@
+   Your directory structure should look similar to the image below.
+
+   .. image:: ../images/windows_1_expected_directory_structure.png
+
+5. Configure Mattermost Server by editing the config.json file at ``c:\mattermost\config\config.json``.  Note that
+   you will need to open the file with Wordpad or another text-editor that supports posix-style line feeds.
+   
+   * Update database name and server in the the connection string:
+     
+     * Old: ``"DataSource": "mmuser:mostest@tcp(dockerhost:3306)/mattermost_test?charset=utf8mb4,utf8"``    
+     * New: ``"DataSource": "mmuser:mmuser_password@tcp(10.0.0.1:3306)/mattermost?charset=utf8mb4,utf8"``
+
+   .. note :: Optionally you may continue to edit configuration settings in ``config.json`` or use the 
+      System Console described in a later section to finish the configuration.
+
+6. Test the Mattermost Server
+
+   a. Open a command prompt, and execute ``cd c:\mattermost\bin`` to change your working directory
+
+   b. Execute ``platform.exe``
+   
+   c. Verify that mattermost is running and connected to the database successfully by confirmed a console 
+      log like ``Server is listening on :8065`` letting you know the service is running.
+
+      .. image:: ../images/windows_2_platform_exe_test.png
+
+   d. Stop the server for now by typing ``ctrl-c``
+
+Configure the Firewall
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. note:: This is not required if you are installing on a single server.
+
+7. Open a command prompt as an administrator
+
+8. Execute the following command to allow inbound traffic to Mattermost
+
+   .. code:: batch
+
+      netsh advfirewall firewall add rule name="Mattermost" dir=in action=allow program="C:\mattermost\bin\platform.exe" enable=yes 
+
+Establish a Windows Service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Establishing a windows service to supervise the Mattermost process is recommended to prevent the need to 
+manually start/stop Mattermost.  The included binary cannot be directly registered as a windows service; therefore, 
+a wrapper-utility must be used.
+
+7. Download the latest version of `NSSM (Non-Sucking Service Manager) <https://nssm.cc/download>`__.
+
+8. From the downloaded zip-file, extract ``win64\nssm.exe`` to ``C:\mattermost\bin\``
+
+9. Open a command line as an administrator
+
+10. To create the windows service, execute the following
+
+    .. code:: batch
+
+       cd c:\mattermost\bin
+       nssm install mattermost c:\mattermost\bin\platform.exe
+
+11. Start the service by executing the following
+
+    .. code:: batch
+
+      net start mattermost
+      tasklist /FI "IMAGENAME eq platform.exe"
+
+Verify Mattermost Connectivity
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To verify all steps executed thus far, we'll attempt to access Mattermost through standard HTTP traffic.  To perform 
+this step, you'll need access to a client machine with a compatible browser (e.g. Safari, Firefox, IE11).
```

Comment:
Is it only these three browsers?

If not, might be better to reference Chrome too, since it's also popular on Windows machines